src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,9 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+# from z3c.form import interfaces
+from zope.i18nmessageid import MessageFactory
+_ = MessageFactory('experimental.safe_html_transform')
```

Comment:
Generally Plone add-ons create one MessageFactory and import that everywhere. For example:-

https://github.com/collective/collective.alias/blob/master/src/collective/alias/interfaces.py#L8

The bobtemplate made this for you, so you just have to do ``from experimental.safe_html_transform import MessageFactory as _`` here.